apps/web/src/common/modules/navigations/gnb/modules/gnb-profile/GNBProfile.vue

```diff
@@ -240,10 +278,20 @@ export default defineComponent<Props>({
              */
             if (!profileMenuRef.contains(target)) hideProfileMenu();
         };
+        const handleClickOutsideCurrencyMenu = (e: PointerEvent) => {
+            const profileMenuRef = state.profileMenuRef;
+            if (!profileMenuRef) return;
+            const target = e.target as HTMLElement;
+            setCurrencyMenuVisible(false);
+            if (!profileMenuRef.contains(target)) hideProfileMenu();
+        };
```

Comment:
description: I tried to merge the language and currency event handlers into one, but it failed for some unknown reason. So, I created two functions 😅 (handleClickOutsideLanguageMenu, handleClickOutsideCurrencyMenu)

Reply:
I'm really curious.